waspc/src/Wasp/Message.hs

```diff
@@ -0,0 +1,5 @@
+module Wasp.Message (MessageType (..), SendMessage) where
```

Comment:
Message is a very general term, so I'd maybe add a header comment here explaining what does it refer to, to give it a bit more context when somebody encounters this piece of code. Or try to find a more specific name if possible.